examples/chacha20poly1305/src/chacha20poly1305.rs

```diff
@@ -7,46 +7,58 @@ use hacspec_poly1305::*;
 
 pub type ChaChaPolyKey = ChaChaKey;
 pub type ChaChaPolyIV = ChaChaIV;
+pub type ByteSeqResult = Result<ByteSeq, u8>;
+
+pub const INVALID_TAG: u8 = 1u8;
```

Comment:
Same, can use a proper enum to describe error cases.

Reply:
I'll change all these to use enums 👍🏻